src/App.tsx

```diff
@@ -1,10 +1,53 @@
+import { FC } from 'react';
+import { TextField, Box, Button } from '@mui/material';
 import './App.css';
+import useConsts from "./utils/consts"
+import { handleChange, handlePaste, handleKeyDown, handleCalculate, handleClear } from './utils/utils';
+
+const App: FC = () => {
+  const { digits, setDigits, lastDigit, setLastDigit, isInputFull, isInputValid, baseUrl } = useConsts();
 
-const App: React.FC = () => {
   return (
     <>
+      <h1>What is my 9th digit?</h1>
+      <Box className="top">
+        {digits.map((digit, index) => (
+          <TextField
+            className="input-box"
+            key={index}
+            id={`digit-${index}`}
+            value={digit}
+            onChange={(e) => handleChange(e, index, digits, setDigits)}
+            onPaste={(e) => handlePaste(e, digits, setDigits)}
+            onKeyDown={(e) => handleKeyDown(e, index, digits, setDigits)}
+          />
+        ))}
+      </Box>
+      <div className="card">
+        <Button
+          id="calculate-btn"
+          variant="contained"
+          disabled={isInputFull}
+          onClick={() => handleCalculate(baseUrl, digits, setLastDigit)}
+          sx={{backgroundColor: "#4d030f", color: "#fcdbe0"}}
```

Comment:
style from styles file

Reply:
these specific fields dont work properly in the styles file

Reply:
of curse they are working with Styles, but never mind u will have plenty of opportunities to check it :)